manifests/t/TexasInstruments/TIConnectCE/6.0.1.344/TexasInstruments.TIConnectCE.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TexasInstruments.TIConnectCE
+PackageVersion: 6.0.1.344
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{3439CFD2-E613-4729-80BB-6770F3C063A4}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/ti-connect/6.0.1/TIConnectCE-6.0.1.344.msi

```

Comment:
This doesn't look like the official TI Link. Please include the PackageUrl in the defaultLocale file